### PR TITLE
Switch to vitest ⚡

### PR DESCRIPTION
**Pull request by adriencaccia**

### CODE REVIEW

File: packages/configuration/src/config/vitest.config.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default 2;
```
Comment (fargito) [inline packages/configuration/src/config/vitest.config.ts:1]: Why is this needed?

**Comment by fargito** [comment]
This is awesome! We need to make this change in swarmion/swarmion as well

File: packages/configuration/src/config/vitest.config.ts (hunk 1 above)
Comment (fargito) [reply packages/configuration/src/config/vitest.config.ts:1]: Maybe the whole `configuration` package is not useful any more

File: packages/serverless-helpers/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="vitest" />
```
Comment (fargito) [inline packages/serverless-helpers/vite.config.ts:1]: I think this file is not needed

File: packages/configuration/src/config/vitest.config.ts (hunk 1 above)
Comment (adriencaccia) [reply packages/configuration/src/config/vitest.config.ts:1]: Yes I agree, we should remove it for now. We can always add it back later

File: packages/serverless-helpers/vite.config.ts (hunk 2 above)
Comment (adriencaccia) [reply packages/serverless-helpers/vite.config.ts]: 👍

**Comment by adriencaccia** [comment]
> This is awesome! We need to make this change in swarmion/swarmion as well

Thanks!

As we discussed, we will merge this right before moving this repo to https://github.com/swarmion/swarmion `examples` folder and deprecating it.

**Review by fargito — APPROVED** [review]

File: backend/core/package.json [hunk 3]
```diff
@@ -19,26 +19,25 @@
...
     "eslint": "^8.18.0",
-    "jest": "^27.5.1",
     "serverless": "^3.19.0",
     "serverless-esbuild": "^1.30.0",
-    "ts-jest": "^27.1.5",
     "ts-node": "^10.8.1",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vite": "^3.0.3",
+    "vitest": "^0.19.1"
```
Comment (CorentinDoue) [inline backend/core/package.json:41]: you don't need the `vite` explicit dependency. You don't need to use `vite` in a backend service

File: backend/core/package.json [hunk 4]
```diff
@@ -19,26 +19,25 @@
...
     "esbuild": "^0.14.47",
     "eslint": "^8.18.0",
-    "jest": "^27.5.1",
     "serverless": "^3.19.0",
     "serverless-esbuild": "^1.30.0",
-    "ts-jest": "^27.1.5",
     "ts-node": "^10.8.1",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vite": "^3.0.3",
```
Comment (CorentinDoue) [inline backend/core/package.json:40]: you don't need the `vite` explicit dependency. You don't need to use `vite` in a backend service

File: backend/core/vite.config.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="vitest" />
+
+import { defineConfig } from 'vite';
```
Comment (CorentinDoue) [inline backend/core/vite.config.ts:3]:
```suggestion
import { defineConfig } from 'vitest/config';
```

File: backend/core/vite.config.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="vitest" />
```
Comment (CorentinDoue) [inline backend/core/vite.config.ts:1]: do you really need this ? I just switch to vitest on one of my project. I didn't need this.
